### PR TITLE
Update gemspec from the generated default

### PR DESCRIPTION
**Pull request by zachallaun**
@davidbalbert I'd love some advice on this.

My current situation

### CODE REVIEW

File: hs.gemspec [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+
   s.add_development_dependency('rake')
-  s.add_development_dependency('rdoc')
-  s.add_development_dependency('aruba')
-  s.add_runtime_dependency('gli','2.5.3')
+  s.add_development_dependency('rspec')
+  s.add_development_dependency('webmock')
+  s.add_development_dependency('vcr')
+
+  s.add_runtime_dependency('gli', '2.5.3')
```
Comment (davidbalbert) [inline hs.gemspec]: You probably want to specify your runtime dependencies with `~>`

File: hs.gemspec [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+  s.add_runtime_dependency('netrc', '0.7.7')
+  s.add_runtime_dependency('git', '1.2.5')
+  s.add_runtime_dependency('octokit', '1.22.0')
+
+  s.require_paths = ['lib']
+  s.files = `git ls-files`.split("\n")
```
Comment (davidbalbert) [inline hs.gemspec:24]: I assume these lines are right, but I don't know where you pulled them from.

Comment (zachallaun) [inline hs.gemspec:24]: davidbalbert/junk :)

**Comment by davidbalbert** [comment]
Also, you should remove and .gitignore your Gemfile.lock. You don't want to have that checked in for gems, only apps. Explanation here: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

**Pull request merged**